spec/rails_helper.rb

```diff
@@ -17,6 +17,7 @@
 # run twice. It is recommended that you do not name files matching this glob to
 # end with _spec.rb. You can configure this pattern with the --pattern
 # option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
+Dir[Rails.root.join('spec/support/*.rb')].each { |f| require f }
```

Comment:
Hmm. It's working for me without this line. I'm pretty sure the glob before also includes files directly inside `spec/support`. Could it be that you have the `--pattern` flag set in your local `.rspec` file? Or perhaps you've been running the specs with spring and need to `spring stop` or run the specs without spring?